File: src/main/java/net/igneo/icv/entity/chestplate/abyssStone/AbyssStoneRenderer.java
package net.igneo.icv.entity.chestplate.abyssStone;

import net.minecraft.client.renderer.entity.EntityRendererProvider;
import software.bernie.geckolib.renderer.GeoEntityRenderer;

public class AbyssStoneRenderer extends GeoEntityRenderer<AbyssStoneEntity> {
    public AbyssStoneRenderer(EntityRendererProvider.Context renderManager) {
        super(renderManager, new AbyssStoneModel());
    }
}

File: src/main/java/net/igneo/icv/entity/chestplate/abyssStone/AbyssStoneRenderer.kt
package net.igneo.icv.entity.chestplate.abyssStone

import net.minecraft.client.renderer.entity.EntityRendererProvider
import software.bernie.geckolib.renderer.GeoEntityRenderer

class AbyssStoneRenderer(renderManager: EntityRendererProvider.Context) :
    GeoEntityRenderer<AbyssStoneEntity?>(renderManager, AbyssStoneModel())
